todo_app/app.py

```diff
@@ -1,35 +1,43 @@
+from importlib.metadata import requires
```

Comment:
I think you can remove this. It's not added as a poetry package, and it doesn't seem to be used?